src/containers/categories-search/CategoriesSearch.jsx

```diff
@@ -0,0 +1,104 @@
+import { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import SearchIcon from '@mui/icons-material/Search'
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward'
+import { Typography, createFilterOptions } from '@mui/material'
+import { styles } from '~/containers/categories-search/CategoriesSearch.styles'
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import HashLink from '~/components/hash-link/HashLink'
+import { authRoutes } from '~/router/constants/authRoutes'
+import { categoryService } from '~/services/category-service'
+import AppAutoComplete from '~/components/app-auto-complete/AppAutoComplete'
+
+const CategoriesSearch = ({ categoryItems, setCategoryItems }) => {
+  const { t } = useTranslation()
+  const [search, setSearch] = useState('')
+  const [allCategories, setAllCategories] = useState(categoryItems)
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await categoryService.getCategories()
```

Comment:
Can we fetch categories with some filters? It is better to filter on backed, not fronted.